packages/design-system/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FunctionComponent } from 'react';
+import AccordionItem, { AccordionItemData } from './AccordionItem';
+
+interface AccordionProps {
+  items: AccordionItemData[];
+  className?: string;
+  bordered: boolean;
+}
+
+const Accordion: FunctionComponent<{
+  items: AccordionItemData[];
+  className?: string;
+  bordered?: boolean;
+}> = ({ items, className, bordered = false }) => {
+  return (
+    <div className={`ds-c-accordion ${className} ${bordered ? 'ds-c-accordion--bordered' : ''}`}>
+      {items.map((i) => (
```

Comment:
Instead of using this pattern of `items`, I think we should move to a model similar to `<Table>`, where the user passes in `AccordianSummary` and `AccordianDetails`, similar to how its implemented here in material ui
https://material-ui.com/components/accordion/

Reply:
While its a little more markup for users, they also can have more control over the accordian markup and styles, i.e. 

The downside is that the markup above works well for accordions that work individually of each other (i.e. multi-select), but not so much for single select. It would require the user to manage state, which may or may not be a bad thing